modules/Hlsjs/Hlsjs.json

```diff
@@ -4,7 +4,10 @@
             "resources/hls.min.js" ,
           "resources/hlsjs.js"
         ],
-        "dependencies": "mw.KBasePlugin",
+        "dependencies": [
+            "mw.KBasePlugin",
+            "mw.Peer5"
```

Comment:
@costa  please remove Peer5 from the default hlsjs dependencies and move it to the peer5 module. - we can't merge this code. thanks,